calico/felix/felix.py

```diff
@@ -242,6 +243,13 @@ def dump_top_level_actors(log):
             _log.warning("Unable to install diag dump handler")
             pass
 
+        # Count cluster existence
+        if config.USAGE_COUNT:
+            calico_version = pkg_resources.require("calico")[0].version
```

Comment:
It is a string.  And tested as working for both the Lambda and in _log statements.

But to avoid any confusion, I'll stringify it.